### ISSUE TITLE
Make Flow-Chart for using the tool

### ISSUE THREAD

**Issue by tedwards2412** [tedwards2412/upper-limit#7]
Make a flow chart to direct people to different chapters based on what there situation is @sliem any good tools for this?

**Comment by sliem** [comment]
Begin prototyping with pen and paper? Anything else seems a bit premature as tools typically constrain you to a shape, and I'm not sure what's the best shape for this decision graph would be given I have very little idea of, e.g., how nested it is. It might be a feature matrix for all I know.

Sidenote: is this a tool? I think of it as a tutorial(s).

**Comment by tedwards2412** [comment]
Yeah thats true, ok I'll just start making one on pen and paper and see how it develops!

Sidenote: Completely agree, not sure why I used the word tool. Definitely more of a tutorial
